roles/setup/handlers/main.yml

```diff
@@ -5,7 +5,9 @@
     state: restarted
     enabled: true
   register: _restart
-  when: not _started.changed # don't restart if we recently started
+  when:
+    - not _started.changed # don't restart if we recently started
+    - receptor_install_method in ['release', 'local', 'package']
 
 - name: Reload Receptor
   ansible.builtin.command: receptorctl --socket {{ receptor_socket_dir }}/{{ receptor_control_filename }} reload
```

Comment:
does this work? receptorctl in this playbook is a bash alias for a particular user and `command` does not have a shell below it. I'm not sure even if `.bashrc` is properly loaded when we use ansible to connect to the node or become either. Pretty sure this handler will fail

Reply:
Great catch, will update to resolve this

Reply:
No longer setting `receptorctl` alias, now using receptor.